### PR TITLE
bt-530: add talent profile screen

### LINKED ISSUE #530: Add Talent Profile screen
I see it as a screen, with top navigation for each tab 
[https://reactnavigation.org/docs/material-top-tab-navigator/](https://reactnavigation.org/docs/material-top-tab-navigator/)

Image

User story

As a Talent, I want to have access to my personal cabinet, so that I can edit my personal info

Acceptance criteria:

1. The content area of page must contain:

- the title “Your profile”
- the tabs “Profile”, “BSA badges”, “Skills and projects”, “CV and contacts”, “Hiring”
- the active buttons “Save”, “Edit” 

2. The content area of the tab “Profile” must contain the fields with the next titles:

- “Profile name”
- “Salary  expectations”
- “Job title”
- “Experience”
- “Current location”
- “Employment type”
- “Briefly tell employers about your experience” 

3. The data that can be entered into the fields “Profile name”,  “Salary  expectations”
“Job title”, “Briefly tell employers about your experience”  should meet the criteria that are written in the [validation document](https://docs.google.com/spreadsheets/d/1OM1GunJPbMa6u0Dn4rm9Cgh32PBfMNOxv588UZYBM_4/edit?usp=sharing).

### PR DESCRIPTION
**Pull request by mskmee**
https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/assets/85788646/ada98cba-c1b1-4ff0-8ffc-ffc9fc16dadf

### CODE REVIEW

File: mobile/src/bundles/common/enums/navigation/talent-onboarding-screen-name.enum.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 const TalentOnboardingScreenName = {
-    PROFILE: 'Profile',
+    PROFILE: 'Profile ',
     BSA_BADGES: 'BSA Badges',
```
Comment (likeri29) [inline mobile/src/bundles/common/enums/navigation/talent-onboarding-screen-name.enum.ts]: Why the empty space?

**Comment by katerynakostikova** [comment]
This screen header should match this UI 
image

File: mobile/src/bundles/talent/components/profile-screen.profile-form/profile-screen-profile-form.tsx [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+    LOCATION_OPTIONS,
+} from './constants/constants';
+import { styles } from './styles';
+
+type Properties = {
+    onSubmit: (payload: ProfileStepDto) => void;
+    userData: ProfileStepDto;
+};
+
+const ProfileScreenProfileForm: React.FC<Properties> = ({
```
Comment (katerynakostikova) [inline mobile/src/bundles/talent/components/profile-screen.profile-form/profile-screen-profile-form.tsx]: Can we reuse forms from onboarding?

**Review by katerynakostikova — CHANGES_REQUESTED** [review]

File: mobile/src/bundles/talent/components/profile-screen.profile-form/profile-screen-profile-form.tsx (hunk 2 above)
Comment (katerynakostikova) [reply mobile/src/bundles/talent/components/profile-screen.profile-form/profile-screen-profile-form.tsx]: They have the same logic as step 1-4, on the each tab screen you just pass initial values to the forms

File: mobile/src/bundles/talent/screens/profile-screen.contacts/profile-screen-contacts.tsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+import { Text, View } from '~/bundles/common/components/components';
+
+const ProfileScreenContacts: React.FC = () => {
+    return (
+        <View>
+            <Text>Contacts</Text>
+        </View>
+    );
+};
```
Comment (katerynakostikova) [inline mobile/src/bundles/talent/screens/profile-screen.contacts/profile-screen-contacts.tsx]: If it is CV and contact form why we don't have it here?

File: mobile/src/bundles/talent/screens/profile-screen.hirings/profile-screen-hirings.tsx [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+import { Text, View } from '~/bundles/common/components/components';
+
+const ProfileScreenHirings: React.FC = () => {
+    return (
+        <View>
+            <Text>Hirings</Text>
+        </View>
+    );
+};
```
Comment (katerynakostikova) [inline mobile/src/bundles/talent/screens/profile-screen.hirings/profile-screen-hirings.tsx]: Let's hide this tab for now I think we'll not finish it till the demo

File: mobile/src/bundles/talent/components/profile-tab-bar/profile-tab-bar.tsx [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+            >
+                <Text category={TextCategory.H3}>Your profile</Text>
+                {!isApproved && <VerificationMessage />}
+                <Pressable onPress={handleLogout}>
+                    <CommunityIcon
+                        name={IconName.LOGOUT}
+                        size={30}
+                        color={Color.TEXT2}
+                    />
+                </Pressable>
```
Comment (katerynakostikova) [inline mobile/src/bundles/talent/components/profile-tab-bar/profile-tab-bar.tsx]: We already merged Logout button component

**Review by katerynakostikova — APPROVED** [review]

**Review by illia-dobro — APPROVED** [review]

**Pull request merged**